source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Feature Spotlight: Accessibility'
+author:
+  name: Brandon Casey
+  github: brandonocasey
+tags:
+  - video.js 6.0
+  - accessibility
+  - a11y
+---
+
+### Accessibility! The most important feature you never knew about.
+
+In the Video.js organization we try hard to have good accessibility. Like most other software, any change can affect the system in unintended ways. For example `MuteToggle` and `VolumeControl` were married into `VolumeMenuButton` in Video.js 5. While this change did allow these controls to work in tandem visually, it also did something unintended. It broke accessibility. In this post we will go over what broke, what the fix was, what accessibility is, and how to test and make sure it works.
+
+> Feel free to skip to the [last section](#The-problem-and-the-solution) if you already know what accessibility is.
+
+### Accessibility? What's that?
+
+Accessible software has support for users with vision, hearing, movement/dexterity, or other impairments. It also helps users that want to use the keyboard to navigate. Out of the box web applications have some accessibility due to the nature of HTML, but this is only the case if you are using native elements in intended ways. If you cannot use native DOM elements, like `<button>`, and instead must use a `<div>` for buttons, then you need worry about accessibility in your page.
+
+Supporting users with hearing impairment is not something that we can do directly for the users of Video.js. Instead we must indirectly support these users by adding support for captions in videos. In Video.js we have had support for captions and subtitles for some time, internally they are called `TextTracks`. In fact Video.js has had support for `WebVTT` format `TextTrack`, which is much more accessible, since version 4.
+
+Supporting users with vision impairment is harder, but fully in our control. To support this group of users our player must be accessible to screen readers. A screen reader is an application that reads elements off of the screen to the user (as the name implies). On top of reading from the screen it also allows the user to interact with the page using only the keyboard or specific gestures on a touchscreen (without using a mouse or needing to directly touch visible items). HTML has certain rules that must be followed so that a page can be accessible. We will go over the basics of these rules in the next section. Screen readers are further supported by having description tracks that can be read out by screen readers during video playback (this is another `TextTrack`).
```

Comment:
"... but _partly_ in our control." (Support for Screen Reader access to controls, etc, is in our control, as is _support_ for description TextTracks, but like captioning, we can't provide the description tracks ourselves).